<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>581</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>940</width>
    <height>581</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>940</width>
    <height>581</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow { background-color:: white}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">QWidget { background-color: rgb(255, 255, 255)}</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>232</width>
      <height>384</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-family:Arial;
	font-size:15px;
	font-weight:bold;
	color: rgb(14, 70, 255);
	padding:6px 24px;
	text-decoration:none;
}</string>
       </property>
       <property name="text">
        <string>Объекты</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="realEstateButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	background-color:#33bdef;
	border-radius:6px;
	border:1px solid #057fd0;
	color:#ffffff;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
	padding:6px 24px;
}
QPushButton:hover {
	background-color:#019ad2;
}
QPushButtonn:active {
	position:relative;
	top:1px;
background-color:#ffffff;
}

        </string>
       </property>
       <property name="text">
        <string>Недвижимое имущество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="movablesButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	background-color:#33bdef;
	border-radius:6px;
	border:1px solid #057fd0;
	color:#ffffff;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
	padding:6px 24px;
}
QPushButton:hover {
	background-color:#019ad2;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

</string>
       </property>
       <property name="text">
        <string>Движимое имущество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="landButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	background-color:#33bdef;
	border-radius:6px;
	border:1px solid #057fd0;
	color:#ffffff;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
	padding:6px 24px;
}
QPushButton:hover {
	background-color:#019ad2;
}
QPushButtonn:active {
	position:relative;
	top:1px;
background-color:#ffffff;
}

        </string>
       </property>
       <property name="text">
        <string>Земельные участки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">QLabel {
font-family:Arial;
	font-size:15px;
	font-weight:bold;
	color: rgb(14, 70, 255);
	padding:6px 24px;
	text-decoration:none;}</string>
       </property>
       <property name="text">
        <string>Ошибки</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="errorButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	background-color:#33bdef;
	border-radius:6px;
	border:1px solid #057fd0;
	color:#ffffff;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
	padding:6px 24px;
}
QPushButton:hover {
	background-color:#019ad2;
}
QPushButtonn:active {
	position:relative;
	top:1px;
background-color:#ffffff;
}

        </string>
       </property>
       <property name="text">
        <string>Выявленные ошибки</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="statButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton  {
	background-color:#33bdef;
	border-radius:6px;
	border:1px solid #057fd0;
	color:#ffffff;
	font-family:Arial;
	font-size:15px;
	font-weight:bold;
	padding:6px 24px;
}
QPushButton:hover {
	background-color:#019ad2;
}
QPushButtonn:active {
	position:relative;
	top:1px;
background-color:#ffffff;
}

        </string>
       </property>
       <property name="text">
        <string>Статистика</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>80</y>
      <width>661</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="objLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QFrame" name="listFrame">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>661</width>
          <height>461</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTableView" name="listObjView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>661</width>
           <height>461</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="objFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>661</width>
          <height>461</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTableView" name="objView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>661</width>
           <height>461</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="ErrorFrame">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>661</width>
          <height>461</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTableView" name="errorTableView">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>661</width>
           <height>461</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="staticticframe">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>661</width>
          <height>461</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QCustomPlot" name="widget_2" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>671</width>
           <height>461</height>
          </rect>
         </property>
        </widget>
       </widget>
       <zorder>staticticframe</zorder>
       <zorder>listFrame</zorder>
       <zorder>objFrame</zorder>
       <zorder>ErrorFrame</zorder>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>911</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Инвентаризация имущества</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
